install/kubernetes/helm/istio/charts/mixer/templates/deployment.yaml

```diff
@@ -1,88 +1,99 @@
+{{- define "mixer_containers" }}
+    spec:
+      serviceAccountName: istio-mixer-service-account
+      volumes:
+      - name: istio-certs
+        secret:
+          secretName: istio.istio-mixer-service-account
+          optional: true
+    {{- if $.Values.nodeSelector }}
+      nodeSelector:
+{{ toYaml $.Values.nodeSelector | indent 8 }}
+    {{- end }}
+      containers:
+      - name: mixer
+        image: "{{ $.Values.global.hub }}/mixer:{{ $.Values.global.tag }}"
+        imagePullPolicy: {{ $.Values.imagePullPolicy }}
+        ports:
+        - containerPort: 9091
+        - containerPort: 9093
+        - containerPort: 42422
+        args:
+          - --configStoreURL=k8s://
+          - --configDefaultNamespace={{ $.Release.Namespace }}
+          - --trace_zipkin_url=http://zipkin:9411/api/v1/spans
+          - --logtostderr
+        resources:
+{{ toYaml $.Values.resources | indent 12 }}
+      - name: istio-proxy
+        image: "{{ $.Values.global.hub }}/{{ $.Values.global.proxy.image }}:{{ $.Values.global.tag }}"
+        imagePullPolicy: {{ $.Values.global.imagePullPolicy }}
+        ports:
+        - containerPort: 15004
+        args:
+        - proxy
+        - mixer
+        - --controlPlaneAuthPolicy
+      {{- if $.Values.global.securityEnabled }}
+        - MUTUAL_TLS
+        - --customConfigFile
+        - /etc/istio/proxy/envoy_mixer_auth.json
+      {{- else }}
+        - NONE #--controlPlaneAuthPolicy
+        - --customConfigFile
+        - /etc/istio/proxy/envoy_mixer.json
+      {{- end }}
+        resources:
+{{ toYaml $.Values.global.proxy.resources | indent 12 }}
+        volumeMounts:
+        - name: istio-certs
+          mountPath: /etc/certs
+          readOnly: true
+{{- end }}
+
+
+{{- if $.Values.splitCheckAndReport }}
+  {{-  $mixers := list "check" "report" }}
+  {{- range $idx, $mname := $mixers }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: istio-mixer-{{ $mname }}
```

Comment:
what was the final decision here (`istio-policy` v. `istio-mixer-check`) ?

Reply:
`istio-policy`, this was still WIP.